percona-server-mongodb-4.2/Dockerfile.k8s

```diff
@@ -99,6 +99,10 @@ RUN set -ex; \
 
 VOLUME ["/data/db"]
 
+RUN set -ex; \
```

Comment:
you need to install it not only for k8s it should be available in official images too, Am I right ?

Reply:
yes, the entrypoint uses this dependency to parse mongodb config file

Reply:
in this case you need to add it to Dockerfile.ubi8 and Dockerfile as well

Reply:
updated